clients/fides-js/src/fides-demo.html

```diff
@@ -35,6 +35,9 @@ <h2>Was the fides global defined?</h2>
 
       <h2>Consent JSON</h2>
       <pre id="consent-json"></pre>
+
+      <h2>Banner Options</h2>
+      <pre id="consent-banner-options"></pre>
```

Comment:
Note that I deleted this duplicate demo site on the latest updates to turborepo branch, so this file got deleted when I merged those changes in